glean-core/src/upload/directory.rs

```diff
@@ -71,7 +73,13 @@ impl PingDirectoryManager {
     ///
     /// Won't panic if unable to delete the file.
     pub fn delete_file(&self, uuid: &str) {
-        let path = self.get_dir().join(uuid);
+        let path = match self.get_file_path(uuid) {
+            Some(path) => path,
+            _ => {
+                log::info!("Cannot find ping file to delete {}", uuid);
```

Comment:
nit: is this an error? should we use `log::error!` instead?

Reply:
Yes, it is an error.